test/Microsoft.Net.Http.Headers.Tests/ContentDispositionHeaderValueTest.cs

```diff
@@ -573,6 +573,21 @@ public void HeaderNamesWithQuotes_ExpectNamesToNotHaveQuote()
             Assert.Equal(expectedFileName, result.FileName);
         }
 
+        [Fact]
+        public void HeaderNamesWithQuotes_CheckSettersToNotHaveQuotes()
+        {
+            var contentDispositionLine = "form-data; name =\"dotnet\"; filename=\"example.png\"";
+            var expectedName = "hello";
+            var expectedFileName = "example.png";
+
+            var result = ContentDispositionHeaderValue.Parse(contentDispositionLine);
+            result.Name = "hello";
+            result.FileName = "\"example.png\"";
```

Comment:
I meant what happens when you set a value that requires quotes, like "my file name with spaces.txt"